src/EVault/modules/Liquidation.sol

```diff
@@ -15,8 +15,8 @@ import "../shared/types/Types.sol";
 abstract contract LiquidationModule is ILiquidation, Base, BalanceUtils, LiquidityUtils {
     using TypesLib for uint256;
 
-    // Maximum liquidation discount that can be awarded under any conditions in wad.
-    uint256 internal constant MAXIMUM_LIQUIDATION_DISCOUNT = 0.2e18;
+    // precision of the discount factor (discount factor scale)
+    uint256 internal constant DFS = 1e18;
```

Comment:
Why do this refactor if we're not planning on making it configurable? It just bloats up the diff size. Also `DFS` is not the best constant name :)